includes/ets_user_qa_function.php

```diff
@@ -404,18 +404,18 @@ public function display_qa_listing_shortcode($atts){
 
 	        $product_id = $atts['product_id'];
 	        if ($product_id !== '') {
-	        	session_start();
-            	$_SESSION['product_id'] = $product_id;
+	        	
         	}
 	    }
 	
 	    if(!empty($product_id)){
 	        $product = wc_get_product($product_id);
 	        if($product){
 	            ob_start();
-	            $this->display_qa_listing();
-	         
-        		return ob_get_clean();
+	            $this->display_qa_listing(); 
+        		$output = ob_get_clean();
+        		$output .= '<div class="ets-shortcode-container"><input type="hidden" name="sh-prd-id" id="sh-product-id" value="' . esc_attr($product_id) . '"></div>';
```

Comment:
If there is no use of wrapper `.ets-shortcode-container` around the hidden field, remove it.